pom.xml

```diff
@@ -489,6 +489,7 @@
 
                 <module>core-kotlin-companion</module>
                 <module>core-kotlin-modules</module>
```

Comment:
Why did we add it there?

Reply:
Because only integration test cases were failing not default